broker/submissions/routes.py

```diff
@@ -16,23 +17,47 @@
 )
 
 
+@submissions_bp.route('/<submission_uuid>/spreadsheet', methods=['POST'])
+def generate_spreadsheet(submission_uuid):
+    submission = app.ingest_api.get_submission_by_uuid(submission_uuid)
+    spreadsheet_job = submission.get('lastSpreadsheetDownloadJob', {}) or {}
```

Comment:
Not sure if 'or {}` is necessary in this assignment expression.

Reply:
Ingest core is populating the lastSpreadsheetDownloadJob field in the json even if it's not in the db. So you can get a submission like the following:
```
{
    ...
    "lastSpreadsheetDownloadJob": null
    ...
}
```